infra/charts/treatment-service/README.md

```diff
@@ -29,23 +29,23 @@ $ helm repo add xp https://turing-ml.github.io/charts
 
 ### Installing the chart
 
-This command will install XP Treatment Service release named `treatment-service` in the `default` namespace.
```

Comment:
You should modify the template file: https://github.com/caraml-dev/xp/blob/main/infra/charts/treatment-service/README.md.gotmpl

And generate the `.md` file from it by using the helm docs tool (see `.pre-commit-config.yaml`)

Reply:
Done! :)